src/Symfony/Bundle/FrameworkBundle/Command/ServerRunCommand.php

```diff
@@ -84,13 +85,14 @@ protected function configure()
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $documentRoot = $input->getOption('docroot');
+        $outputStyle = new SymfonyStyle($input, $output);
```

Comment:
Here we call this variable `$outputStyle` and in the rest of the commands we just call it `$output`. I'm not sure which is the best strategy to follow. @stof already expressed his concerns about the name of this variable in other pull request. In any case, I agree that it's hard to name this variable properly.